### PR TITLE
add mummah

### PR DESCRIPTION
**Pull request by nuclearkatie**
I added myself to the space next to where Alex will be & used the blank hacker image

### CODE REVIEW

File: source/people/mummah.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+              :width: 32
+.. _tw_icon: https://twitter.com/nuclearkatie
+
+.. |li_icon| image:: linkedin.png
+              :width: 32
+.. _li_icon: https://www.linkedin.com/in/nuclearkatie
+
+.. |wb_icon| image:: website-logo.png
+              :width: 32
+.. _wb_icon: https://nuclearkatie.com
```
Comment (gonuke) [inline source/people/mummah.rst]: This link was failing for me, but seemed to work with just `http` (rather than `https`)

**Comment by gonuke** [comment]
Looks good - made a couple of minor comments.

**Comment by nuclearkatie** [comment]
Good catch. I addressed both comments in the new commit

**Comment by gonuke** [comment]
LGTM - thanks @nuclearkatie

**Pull request merged**